src/transform.jl

```diff
@@ -76,6 +76,7 @@ julia> name_only(:(x::Int))
 function name_only(@nospecialize(ex))
     ex isa Symbol && return ex
     ex isa QuoteNode && return ex.value
+    ex isa JLField && return ex.name
```

Comment:
I'm wondering why would you need JLField here it won't appear in your Julia expression?

Reply:
True, I would like some functions to apply transformations both to `Expr` and to structured expressions. What do you think?

Reply:
Ah I can see why - I guess you just use `Expr` to store arbitrary elements in it, since the type inside `Expr` is `Any` so that's actually allowed... yeah I think it doesn't hurt to have that, but then we will also need to support `JLKwField`, `JLStruct`, `JLKwStruct` here